source/beamline/CLS/CLSSIS3820ScalerContinuousMode.h

```diff
@@ -1,31 +1,25 @@
-#ifndef CLSSIS3820SCALERMODECONTROL_H
-#define CLSSIS3820SCALERMODECONTROL_H
+#ifndef CLSSIS3820SCALERCONTINUOUSMODE_H
+#define CLSSIS3820SCALERCONTINUOUSMODE_H
 
-#include "beamline/AMPseudoMotorControl.h"
+#include "beamline/AMEnumeratedControl.h"
 
-class CLSSIS3820ScalerModeControl : public AMPseudoMotorControl
+class CLSSIS3820Scaler;
+
+class CLSSIS3820ScalerContinuousMode : public AMEnumeratedControl
```

Comment:
And now that I've read through the pull request myself, there is also a "Scan Mode".  My suggestion is "Acquisition Mode" since it distinguishes it from Scan and Input mode sufficiently.